README.md

```diff
@@ -54,21 +80,39 @@ cargo build --release --target=<your-rust-target>
 
 ### Database
 
-In order to use Simbiota, you need a detection database.
-Download one from our [`database-releases`](https://github.com/simbiota/database-releases/releases) page.
+**In order to use Simbiota, you need a detection database.**
+You may either use `simbiota-update.timer` systemd timer to download the latest database release every day.
+
+Or update the database with our provided script `/usr/sbin/simbiota-update.sh` (first read it, then run it as root).
+
+Or download one from our [`database-releases`](https://github.com/simbiota/database-releases/releases) page.
 ```bash
-curl -L https://github.com/simbiota/database-releases/releases/download/20230630/simbiota-arm-20230630.sdb -o /var/lib/simbiota/simbiota-arm-20230630.sdb
+curl -L https://github.com/simbiota/database-releases/releases/download/20230630/simbiota-arm-20230630.sdb -o /var/lib/simbiota/database.sdb
```

Comment:
IMO we should replace the exact link with a placeholder one, otherwise users will download an old database